cli/src/commands/extensions/mod.rs

```diff
@@ -26,6 +34,9 @@ pub fn command<'a>() -> Command<'a> {
         .subcommand(
             Command::new("remove").about("Uninstall extension").arg(arg!([NAME]).required(true)),
         )
+        .subcommand(
+            Command::new("new").about("Create a new extension").arg(arg!([PATH]).required(true)),
```

Comment:
I think it would be good UX to also allow users to launch `phylum extension new` without an argument and have that create an extension in the current directory, similarly to what other ecosystem tools do.

But I'd say it's not super urgent and we could tackle this in a separate issue/PR.

Reply:
I've decided to mimic `cargo new` in behavior and honestly think that's a good idea. There's already enough invariants to ensure and I think creating one in the current directory would just come with unnecessary ambiguity.